### ISSUE TITLE
Biological evaluation 2021

### ISSUE THREAD

**Issue by dmitrij176** [OpenSourceMycetoma/Series-1-Fenarimols#57]

**Comment by dmitrij176** [comment]
![Biological evaluation (June 2021)](https://user-images.githubusercontent.com/44785501/125297126-a2d88900-e32f-11eb-8816-39e33f1513cb.png)


Good day @wwjvdsande. UCL compounds have been delivered to Rotterdam today. Due to Brexit and corresponding change in delivery terms to the EU, we had some issues with customs which caused an unexpected delay (the shipment was originally scheduled for delivery on the 7th of July). But now we know how to deal with that issue properly, so should be fine in the future.
The parcel contains 10 compounds (8 Fenarimol analogues and 2 oximes). All supplementary information (including a separate table with exact masses, SMILES, MW etc) you will find in a paper copy within the package.
Please let me know if you need anything else.
Regards,
Dmitrij

**Comment by wwjvdsande** [comment]
@dmitrij176 we just received in 30 minutes ago. Is it OK to store them at room temperature before we dilute in DMSO? The compouds of @kym834 are suspected to arrive this week as well so than we can test them all together in one go. After adding DMSO we will store them at -20 or do you prefer otherwise?

**Comment by dmitrij176** [comment]
> @dmitrij176 we just received in 30 minutes ago. Is it OK to store them at room temperature before we dilute in DMSO? The compouds of @kym834 are suspected to arrive this week as well so than we can test them all together in one go. After adding DMSO we will store them at -20 or do you prefer otherwise?

They are all stable at room temperature. Following synthesis and purification, we usually store them in a freezer, so both options are ok. Also, I did some NMR characterisation using deuterated DMSO as a solvent, and had no stability problems as well.

**Comment by MFernflower** [comment]
@wwjvdsande once all the compounds arrive - are you thinking you will be able to screen the potent ones in waxworms?

**Comment by wwjvdsande** [comment]
@MFernflower we will test the compounds first in vitro. We will use the same criteria as for the previous compounds. Only those compounds who show potent in vitro activity will be tested in the larvae. The larvae are labour intensive and there is no use in screening compounds which have no effect on the fungus in vitro.

**Comment by dmitrij176** [comment]
@mattodd @wwjvdsande having started new oxime series, I came across a small issue with MyOS repository. The Google excel sheet where we register our new project molecules appears under the name ''MYOS compounds (Fenarimol analogues)''. So the implication is that the table is specifically designated for Fenarimols. But what happens when we start exploring new classes as in my case?
Ideally, all entries should be in the same table to avoid ambuguity and confusion, because if we start new excel pages for other classes then we might end up with coding errors again. 
Would renaming the table to just MyOS General be an option? My latest shipment has 2 ketoximes for biological evaluation, but the structures currently are not assigned with unique codes.

**Comment by wwjvdsande** [comment]
@dmitrij176 @mattodd I think we could rename it to MyOS compounds but there should be a column with an identifier which locates it to a molecule series so that for us non-chemists it makes it possible to select different compound series for further analyses.

**Comment by mattodd** [comment]
Yes, rename the sheet, add a column for Series. I would strongly recomment Fenarimols being Series 1 (i.e. a "1" for those entries) and then the other series being given unambiguous names that are added to the front page of any associated repositories. I believe 2 is aminothiazoles, 3 is "tricyclics" (though we need to be careful that that is accurately defined) and so on. Looks like "ketoximes" would be Series 6?

**Comment by dmitrij176** [comment]
> @dmitrij176 @mattodd I think we could rename it to MyOS compounds but there should be a column with an identifier which locates it to a molecule series so that for us non-chemists it makes it possible to select different compound series for further analyses.

@wwjvdsande do you think it would be better to have all molecules in one table or diversify them based on their respective classes?
Another option might be adding all entries to one table, but then create separate excel files (like we did for in vitro/vivo and logD) where we copy specific information related to that class. Maybe that could make it easier for evaluation purposes.

**Comment by wwjvdsande** [comment]
@dmitrij176 @mattodd I agree with Matt to add the series number and keep it consistent. For me it is OK if everything stays in one table. Might be easier. In that case anyone can substract the data they need in their own software.

**Comment by dmitrij176** [comment]
![Biological evaluation (June 2021)](https://user-images.githubusercontent.com/44785501/125426132-8877675d-dcab-4bca-be08-c0ec5198aab9.png)


@wwjvdsande updated version

**Comment by MFernflower** [comment]
@mattodd series 3 should really be called phenothiazines - I can update repo with new research papers etc

**Comment by dmitrij176** [comment]
Hi @MA-Jjingyi. Wanted to ask how is the _in vitro_ screening going? Do you know when the results are expected for release?

**Comment by dmitrij176** [comment]
@wwjvdsande @MA-Jjingyi has MYOS_00016_00_01 (DM7-1 active with piperidine motif) ever been tested against  F. senegalensis?

**Comment by MA-Jjingyi** [comment]
@dmitrij176 sorry to slowly reply.  we have tested the compounds against F.S and M.M at 25uM. it works well with M.M, not F.S. but we just tested one time in the past. I will repeat the test this week. and I will let you know the results soon.

**Comment by MA-Jjingyi** [comment]
@dmitrij176  the compound named MYOS_00016_00_01 (DM7-1)just inhibitor F.senegalensis growth at 100uM from 3 times experiments, not 25uM.
